src/tracing/cls.js

```diff
@@ -0,0 +1,138 @@
+'use strict';
+
+var tracingUtil = require('./tracingUtil');
+var hooked = require('cls-hooked');
+var currentSpanKey = 'csKey';
+
+var exitSpans = ['node.http.client', 'elasticsearch', 'mongo', 'mysql'];
```

Comment:
My intention was that instrumentations will set it themselves and that we therefore do not need such a list. Is this not possible?

Reply:
I did it this way in Ruby which is why I made an assumption.  I suppose we could instead add the `span.k` assignment in each instrumentation code directly but the  intention was to simplify instrumentation by moving all the span creation details into CLS.

If you have a strong preference I can change it.  Otherwise let's leave it and I'll revisit it on my next run through the code.  lmk